pyrobosim/pyrobosim/planning/pddlstream/utils.py

```diff
@@ -132,14 +132,23 @@ def process_goal_specification(goal_literals, world):
     :param goal_literals: List of literals describing a goal specification.
     :type goal_literals: list[tuple]
     """
+
+    def process_element(elem):
+        """Helper function to process individual elements recursively"""
```

Comment:
Need to add description of what the input and output arguments are. See other docstrings.